### PR TITLE
Added the hugoExtended

### PR DESCRIPTION
**Pull request by furyfire**
Added flag that allows to instead download the extended hugo binary

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "go.formatTool": "goimports"
```
Comment (xoxys) [inline .vscode/settings.json]: Please remove your editors settings folder

**Comment by cbrgm** [comment]
LGTM, thanks

**Comment by furyfire** [comment]
Fixed.

**Comment by xoxys** [comment]
@furyfire Don't clone a PR before merged :)

File: main.go [hunk 2]
```diff
@@ -92,6 +92,11 @@ func main() {
 			EnvVar: "PLUGIN_HUGO_VERSION",
 			Value:  "",
 		},
+		cli.BoolFlag{
+			Name:   "hugoExtended",
+			Usage:  "If the hugo extended package should be used",
+			EnvVar: "PLUGIN_HUGO_EXTENDED",
```
Comment (tboerger) [inline main.go]: Maybe just `PLUGIN_EXTENDED`? It's already obvious enough that you are using hugo with something like that:

```
- name: deploy
  image: plugins/hugo
  settings:
    extended: true
```

**Comment by marcotuna** [comment]
Hello, when does this going to be merged?

I am having issues because the extended is not implemented yet.

```
Building sites … ERROR 2019/07/04 10:21:27 error: failed to transform resource: TOCSS: failed to transform "scss/style.scss" (text/x-scss): this feature is not available in your current Hugo version, see https://goo.gl/YMrWcn for more information
```

**Comment by tboerger** [comment]
No reaction to my change request, so who knows.

**Comment by xoxys** [comment]
@tboerger Fixed in https://github.com/drone-plugins/drone-hugo/pull/19. Please review. Would be good to have this feature :)

**Comment by tboerger** [comment]
Closing this PR in favor of https://github.com/drone-plugins/drone-hugo/pull/19

**Pull request closed without merging**